.github/workflows/translate.yml

```diff
@@ -0,0 +1,61 @@
+name: "Update translations"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 3 * * *"
+    
+permissions: {}
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  update-translations:
+    permissions:
+      # for git push
+      contents: write
+    runs-on: ubuntu-latest
+
+    env:
+      TX_TOKEN: ${{ secrets.TX_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          ref: ${{ github.ref }}
+
+      - name: l10n-push
+        if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+        # we need to use a different docker image for those two, this appears to be a bit tricky with github actions out of the box
+        run:  bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "cd ownCloud/Resources/en.lproj && tx push -s --skip"
+
+      - name: l10n-pull
+        run: bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "cd ownCloud/Resources/ && tx pull --force --skip --all"
+
+      - name: update-repo-before-commit
+        run: |
+          git status
+          git add ownCloud/Resources/*.strings
+          git status
+          git stash
```

Comment:
Please either reintroduce the pull (the repo might have changed in the last couple of minutes) or rename the step.

Reply:
fixed